### ISSUE TITLE
Progress LIRADeconvolver.run

### ISSUE THREAD

**Issue by Borlaff** [astrostat/pylira#67]
Hi, 

Would it be possible to add some kind of progress tracker for LIRADeconvolver.run? This might not be easy since it is calling the R/cpp _lira modules, but I was wondering if there might be a way to know how many simulations have been already produced. This might be particularly useful for runs with large number of pixels (in a M1 laptop it takes about 3 hours to run 200 simulations on a 1024x1024 pixel map). 

Thank you!
Alex

**Comment by adonath** [comment]
I agree this would be useful. As far as I know this would be possible to implement using pybind11's callback mechanism. But right now I don't have time to look into it.
